src/style.css

```diff
@@ -0,0 +1,37 @@
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```

Comment:
not all the stars need `margin` 😉

<img width="203" alt="image" src="https://user-images.githubusercontent.com/55802630/172790319-9db556f6-2ef6-4fb5-9a71-a7411b004eb0.png">